test/models/page.test.js

```diff
@@ -0,0 +1,145 @@
+var chai = require('chai')
+  , expect = chai.expect
+  , sinon = require('sinon')
+  , sinonChai = require('sinon-chai')
+  , proxyquire = require('proxyquire')
+  ;
+chai.use(sinonChai);
```

Comment:
For me, the test utilities of node-mongodb-native(https://github.com/mongodb/node-mongodb-native/tree/2.0/test) looks good because they setups a test environment witout exposing the logic and libraries used in tests to test cases.